app/tasks/bookmark_save_task.py

```
@@ -1,229 +1,379 @@
+
+    return total_deleted
+
+
+async def _save_content_chunks(session, bookmark_data: BookmarkData, body_text: str,
+                               similar_bookmarks: List[Dict]) -> tuple:
+    """콘텐츠 청크들을 저장합니다."""
+    metadata = {
+        "memo": bookmark_data.memo,
+        "summary": bookmark_data.summary
+    }
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)